rest/server_context.go

```diff
@@ -549,16 +549,21 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		}
 	}
 
-	if config.RevsLimit != nil && *config.RevsLimit > 0 {
+	if config.RevsLimit != nil {
```

Comment:
What about moving this code into a hypothetical `dbcontext.SetRevsLimit()`?  And this code would just propagate the error back to the caller .. eg:

```
if err := dbcontext.SetRevsLimit(config.RevsLimit); err != nil {
   return nil, err
}
```

Reply:
My only concern is that it would obscure the dependency on dbcontext.AllowConflicts(), which might lead to a future regression where we try to call SetRevsLimit before setting AllowConflicts().

We could add that as a comment in getOrAddDatabaseFromConfig, though.

Reply:
In that case, let's leave it as-is for now until we can come up with a way to avoid that ordering issue.. it's probably safer to depend on having code immediately apparent than on doc comments which tend to get lost/moved.